src/library/scala/Option.scala

```diff
@@ -63,6 +63,27 @@ object Option {
  *  This allows for sophisticated chaining of $option values without
  *  having to check for the existence of a value.
  *
+ * These are useful helper functions that exist for both $some and $none.
+ *  - [[isDefined]] — True if not empty
+ *  - [[isEmpty]] — True if empty
+ *  - [[nonEmpty]] — True if not empty
+ *  - [[orElse]] — Return default optional value if empty
+ *  - [[getOrElse]] — Return default value if empty
```

Comment:
in this and the previous line, perhaps "Compute and return" instead of "Return"?

Reply:
Good point, since the expression gets lazy evaluated.